mongo/tests/conftest.py

```diff
@@ -101,8 +103,8 @@ def instance_1valid_and_1invalid_custom_queries():
 
 
 @pytest.fixture
-def check():
-    return MongoDb('mongo', {}, {})
+def check(instance):
+    return lambda instance: MongoDb('mongo', {}, {}, [instance])
```

Comment:
```suggestion
    return lambda instance: MongoDb('mongo', {}, [instance])
```

https://github.com/DataDog/integrations-core/blob/6654a9510ab067bbc0ffa03b387b8841d60c1bc2/datadog_checks_base/datadog_checks/base/checks/base.py#L126-L129

Reply:
Not sure if there much value having `check` as fixture instead of just calling `MongoDb('mongo', {}, [instance])` in tests.

Reply:
Mongo still has the old signature: https://github.com/DataDog/integrations-core/blob/master/mongo/datadog_checks/mongo/mongo.py#L86

Which is getting changed in this other PR that this one is a split off: https://github.com/DataDog/integrations-core/pull/6085

The idea of the fixture is that when I change the signature I only need to edit the fixture instead of every test

Reply:
Fixture useful to avoid imports in every file & cleans up code